apps/mobile/src/components/register-adoption-form/compose/form-steps/form-steps.tsx

```diff
@@ -24,6 +24,7 @@ function PetNameStep() {
       input={{
         ...commonInputProperties,
         placeholder: t('REGISTER_ADOPTION.FORM.NAME_PLACEHOLDER'),
+        testID: 'adoption-form-name-input',
```

Comment:
Are these testID needed? I can't find them in the code, only `adoption-form-age-input` seems to be used

Reply:
Yes, check `fillOutTextInput` function in `adoption-form.component.test.tsx` file.